simpinkscr/svg_to_simp_ink_script.py

```diff
@@ -1022,9 +1022,15 @@ def convert_guide(self, node):
         if color is not None:
             extra = ', color=%s' % repr(color)
 
+        # Determine if we were given a label.
+        extra_label = ''
+        label = node.get('inkscape:label')
+        if label is not None:
+            extra_label = ', label=%s' % repr(label)
+
         # Generate code and wrap it in a statement.
         code = ['guides.append(guide((%.10g, %.10g), %.10g%s))' %
-                (pos[0], pos[1], angle, extra)]
+                (pos[0], pos[1], angle, extra, extra_label)]
```

Comment:
This doesn't work because the format string represents four arguments while you're passing in five.  However, if you append the label code to `extra` and revert to `(pos[0], pos[1], angle, extra)` here, everything should work.

Reply:
You are right. This should be solved now since `extra_label` was dropped.